src/index.html

```diff
@@ -13,5 +13,110 @@
   <body>
     <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset>
+        <legend>Personal information</legend>
+        <div>
+          <label>
+            Surname:
+            <input name="Surname"
+            type="text"
+            autocomplete="off">
```

Comment:
```suggestion
            <input
              name="Surname"
              type="text"
              autocomplete="off"
            >
```
[checklist p.3](https://github.com/mate-academy/layout_html-form/blob/master/checklist.md)

fix it all over the page